app/views/users/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<% provide :title, t(".sign_up") %>
+<h1><%= t ".sign_up" %></h1>
+<%= render "shared/flash_message" %>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: signup_path do |f| %>
```

Comment:
form for là gì? ngoài form for có những loại form nào nữa e

Reply:
form for được dùng khi có một đối tượng. Nó thường được sử dụng khi tạo mới hay cập nhất một đối tượng. 
Ngoài form for thì còn có form tag và form with, form tag dùng khi không có đối tượng còn form with dùng được cho cả khi không có đối tượng hoặc có đối tượng